vpr/src/place/move_utils.h

```diff
@@ -4,6 +4,12 @@
 #include "move_transactions.h"
 #include "compressed_grid.h"
 
+//Run-time flag to control when placer debug information is printed
+//Note : only enables debug output if compiled with VTR_ENABLE_DEBUG_LOGGING is defined
+#ifdef VTR_ENABLE_DEBUG_LOGGING
```

Comment:
This f_placer_debug variable seems to be used as some kind of read/write control variable later on.  Needs a better comment as I'm not sure what it's really being used for.  If you don't need a global variable, better to get rid of it (not clear from its use that it actually needs to be global).